release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2154,6 +2154,11 @@ To make the required change, follow the instructions in link:https://access.redh
 
 * In hosted control planes for {product-title}, on the bare metal and {VirtProductName} platforms, the auto-repair function is disabled. (link:https://issues.redhat.com/browse/OCPBUGS-20028[*OCPBUGS-20028*])
 
+* Agent-based installations on vSphere will fail due to a failure to remove node taint, which causes the installation to be stuck in a pending state.
+{sno-caps} clusters are not impacted.
+There is no current workaround.
+(link:https://issues.redhat.com/browse/OCPBUGS-20049[*OCPBUGS-20049*])
```

Comment:
Did you intend these line breaks to render as new lines? If not, suggest removing the hard breaks.

Reply:
These are all meant to be parts of a single paragraph, I'm just splitting them up in the adoc for a cleaner look/better ability to reference specific sentences.